packages/eas-cli/src/build/configure.ts

```diff
@@ -54,18 +54,18 @@ const EAS_JSON_MANAGED_DEFAULT: EasJson = {
       developmentClient: true,
       distribution: 'internal',
       ios: {
-        resourceClass: ResourceClass.M1_MEDIUM,
+        resourceClass: ResourceClass.M_MEDIUM,
```

Comment:
didn't we want to remove those defaults? Or do we want to remove them, but not just yet?

Reply:
They will be removed as a part of https://github.com/expo/eas-cli/pull/1734